ODIN_II/regression_test/benchmark/operators/pca_1_always_block_2_edge_input

```diff
@@ -0,0 +1,9 @@
+GLOBAL_SIM_BASE_CLK reset set a
+0 0 0 0
```

Comment:
The first two lines look like the only case for deassign. What happens when you set all the inputs back to zero at the end? What happens when you deassign after assign for each individual case (if reset, if set)?

e.g. 
...
0 1 1 1 <- assign q = 1'b0
0 0 0 0 <- deassign - what does this output? (should still be 0 - is this the case?)
0 0 1 0 <- assign q = 1'b1
0 0 0 1 <- deassign - what does this output? (should still be 1 - is this the case?)